src/main/java/com/oltpbenchmark/distributions/UniformGenerator.java

```diff
@@ -0,0 +1,38 @@
+package com.oltpbenchmark.distributions;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+public class UniformGenerator extends IntegerGenerator {
+  int min, max;
```

Comment:
I realize this is trivial but split these two members across separate lines to match the coding style of other classes.

Reply:
Done!